Sources/Controllers/2.0/BottomSheet/OABaseBottomSheetViewController.h

```diff
@@ -8,6 +8,12 @@
 
 #import <UIKit/UIKit.h>
 
+typedef NS_ENUM(NSInteger, EOAScrollableMenuState)
```

Comment:
This is wrong. Do not expose internal state. Use methods goFullScreen/goExpanded/goMinimized instead.